src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2020_11_01/models/_models.py

```diff
@@ -0,0 +1,2604 @@
+     nodes in agent pool.
+    :type node_labels: dict[str, str]
+    :param node_taints: Taints added to new nodes during node pool create and
+     scale. For example, key=value:NoSchedule.
+    :type node_taints: list[str]
+    :param proximity_placement_group_id: The ID for Proximity Placement Group.
+    :type proximity_placement_group_id: str
+    """
+
+    _validation = {
+        'id': {'readonly': True},
+        'name': {'readonly': True},
+        'type': {'readonly': True},
+        'node_image_version': {'readonly': True},
+        'provisioning_state': {'readonly': True},
+        'power_state': {'readonly': True},
+    }
+
+    _attribute_map = {
+        'id': {'key': 'id', 'type': 'str'},
+        'name': {'key': 'name', 'type': 'str'},
+        'type': {'key': 'type', 'type': 'str'},
+        'count': {'key': 'properties.count', 'type': 'int'},
+        'vm_size': {'key': 'properties.vmSize', 'type': 'str'},
+        'os_disk_size_gb': {'key': 'properties.osDiskSizeGB', 'type': 'int'},
+        'os_disk_type': {'key': 'properties.osDiskType', 'type': 'str'},
+        'vnet_subnet_id': {'key': 'properties.vnetSubnetID', 'type': 'str'},
+        'pod_subnet_id': {'key': 'properties.podSubnetID', 'type': 'str'},
+        'max_pods': {'key': 'properties.maxPods', 'type': 'int'},
+        'os_type': {'key': 'properties.osType', 'type': 'str'},
+        'max_count': {'key': 'properties.maxCount', 'type': 'int'},
+        'min_count': {'key': 'properties.minCount', 'type': 'int'},
+        'enable_auto_scaling': {'key': 'properties.enableAutoScaling', 'type': 'bool'},
+        'agent_pool_type': {'key': 'properties.type', 'type': 'str'},
+        'mode': {'key': 'properties.mode', 'type': 'str'},
+        'orchestrator_version': {'key': 'properties.orchestratorVersion', 'type': 'str'},
+        'node_image_version': {'key': 'properties.nodeImageVersion', 'type': 'str'},
+        'upgrade_settings': {'key': 'properties.upgradeSettings', 'type': 'AgentPoolUpgradeSettings'},
+        'provisioning_state': {'key': 'properties.provisioningState', 'type': 'str'},
+        'power_state': {'key': 'properties.powerState', 'type': 'PowerState'},
+        'availability_zones': {'key': 'properties.availabilityZones', 'type': '[str]'},
+        'enable_node_public_ip': {'key': 'properties.enableNodePublicIP', 'type': 'bool'},
+        'scale_set_priority': {'key': 'properties.scaleSetPriority', 'type': 'str'},
+        'scale_set_eviction_policy': {'key': 'properties.scaleSetEvictionPolicy', 'type': 'str'},
+        'spot_max_price': {'key': 'properties.spotMaxPrice', 'type': 'float'},
+        'tags': {'key': 'properties.tags', 'type': '{str}'},
+        'node_labels': {'key': 'properties.nodeLabels', 'type': '{str}'},
+        'node_taints': {'key': 'properties.nodeTaints', 'type': '[str]'},
+        'proximity_placement_group_id': {'key': 'properties.proximityPlacementGroupID', 'type': 'str'},
```

Comment:
I don't see my properties here, is this not generated based on latest dev branch?

Reply:
I updated the sdk from the latest generated artifacts, should have your changes now!